Makefile

```diff
@@ -18,6 +18,8 @@ ROOT_DIR_RELATIVE := .
 
 include $(ROOT_DIR_RELATIVE)/common.mk
 
+GO_VERSION ?= 1.20.10
```

Comment:
May be a follow up PR required for maintaining the golang version in one place and use it everywhere including dockerfiles, makefile and GitHub action